pipelines/common/src/main/java/com/google/fhir/analytics/ParquetUtil.java

```diff
@@ -303,15 +314,15 @@ private synchronized void flush(String resourceType) throws IOException, Profile
     WriterWithCache writer = writerMap.get(resourceType);
     if (writer != null && writer.getDataSize() > 0) {
       writer.close();
-      writerMap.remove(resourceType);
+      createWriter(resourceType, null);
     }
   }
 
   private synchronized void flushViewWriter(String viewName) throws IOException, ProfileException {
     ParquetWriter<GenericRecord> writer = viewWriterMap.get(viewName);
     if (writer != null && writer.getDataSize() > 0) {
       writer.close();
-      viewWriterMap.remove(viewName);
+      createWriter(viewName, viewMap.get(viewName));
```

Comment:
Please add a TODO comment here to investigate why we need to create the writer here. We don't need to block this PR on this but I like us to understand why not creating the writer here breaks our tests (unless if you already know why, please let me know).

Reply:
Done.